site/content/en/docs/manual/advanced/shortcuts.md

```diff
@@ -2,79 +2,57 @@
-|                            | _Attribute annotation mode_                                                                              |
-| `Up Arrow`                 | Go to the next attribute (up)                                                                            |
-| `Down Arrow`               | Go to the next attribute (down)                                                                          |
-| `Tab`                      | Go to the next annotated object in current frame                                                         |
-| `Shift+Tab`                | Go to the previous annotated object in current frame                                                     |
-| `<number>`                 | Assign a corresponding value to the current attribute                                                    |
-|                            | _Standard 3d mode_                                                                                       |
-| `Shift+Up Arrow`           | Increases camera roll angle                                                                              |
-| `Shift+Down Arrow`         | Decreases camera roll angle                                                                              |
-| `Shift+Left Arrow`         | Decreases camera pitch angle                                                                             |
-| `Shift+Right Arrow`        | Increases camera pitch angle                                                                             |
-| `Alt+O`                    | Move the camera up                                                                                       |
-| `Alt+U`                    | Move the camera down                                                                                     |
-| `Alt+J`                    | Move the camera left                                                                                     |
-| `Alt+L`                    | Move the camera right                                                                                    |
-| `Alt+I`                    | Performs zoom in                                                                                         |
-| `Alt+K`                    | Performs zoom out                                                                                        |
+These shortcuts are organized by scopes. Some are global, meaning they work across the entire application, while others are specific to certain sections or workspaces. This setup allows for shortcut reuse in different scopes, depending on whether they might conflict. For example, global shortcuts must be unique since they apply across all pages and workspaces. However, similar shortcuts can be used in different workspaces, like having the same shortcuts in both the Standard Workspace and the Standard 3D Workspace, as these two do not coexist.
```

Comment:
```suggestion
These shortcuts are organized by scopes. Some are global, meaning they work across the entire application, while others are specific to certain sections or workspaces. This approach allows reusing the same shortcuts in different scopes, depending on whether they might conflict. For example, global shortcuts must be unique since they apply across all pages and workspaces. However, similar shortcuts can be used in different workspaces, like having the same shortcuts in both the Standard Workspace and the Standard 3D Workspace, as these two do not coexist.
```